code/game/turfs/closed/_closed.dm

```diff
@@ -38,6 +38,12 @@
 /turf/closed/indestructible/singularity_act()
 	return
 
+/turf/closed/indestructible/attackby(obj/item/attacking_item, mob/user, params)
+	..()
+
+	if(istype(attacking_item, /obj/item/poster) && Adjacent(user))
+		return place_poster(attacking_item, user)
```

Comment:
should we `return ..()` instead or is the rest of the attackby code important for reasons beyond my comprehension

Reply:
Do you mean putting return ..() after the poster check? Also I don't think the earlier attackby code is important in this context.

Reply:
yes i meant after. sorry for getting back to you so late